### PR TITLE
including applicant2approved state to about to submit with validation…

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-723

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java [hunk 1]
```diff
@@ -62,4 +65,20 @@ public void shouldPassValidationAndGiveSuccessWhenHelpWithFeesApplied() throws I
...
+        request.put("marriageDate", LocalDate.now().minus(1, YEARS).minus(1, DAYS));
+
+        Response response = triggerCallback(request, CITIZEN_SUBMIT, ABOUT_TO_SUBMIT_URL);
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+
+        // marriageDate and payments.id are ignored using ${json-unit.ignore}
+        // assertion will fail if the above elements are missing actual value
+        assertThatJson(response.asString())
+            .when(IGNORING_EXTRA_FIELDS)
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java:80]: better to use TREATING_NULL_AS_ABSENT so we just ignore null fields

File: src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java [hunk 2]
```diff
@@ -231,6 +232,16 @@ public static CaseData validApplicant1CaseDataMap() {
         return caseData;
     }
 
+    public static CaseData validApplicant2CaseDataMap() {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java:235]: it should be valid case data as it is not a map

File: src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java [hunk 3]
```diff
@@ -231,6 +232,16 @@ public static CaseData validApplicant1CaseDataMap() {
...
+        CaseData caseData = validApplicant1CaseDataMap();
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/testutil/TestDataHelper.java:236]: validApplicant1CaseDataMap returns case data so named incorrectly :)

File: src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java (hunk 1 above)
Comment (velimd) [reply src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java:80]: Alot of the functional test have this, might need another pr to change it all as i change them it breaks. probably to do with extra fields that get added.

**Pull request closed without merging**